emission/export/export.py

```diff
@@ -0,0 +1,51 @@
+import logging
+logging.basicConfig(level=logging.DEBUG)
+import gzip
+import os 
+
+import uuid
+import json
+import bson.json_util as bju
+import emission.storage.timeseries.timequery as estt
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.timeseries.cache_series as estcs
+import emission.net.usercache.abstract_usercache as enua
+def export(user_id, ts, start_ts, end_ts, file_name, ma_bool):
+    if ma_bool:
+        ma_time_query = estt.TimeQuery("metadata.write_ts", start_ts, end_ts)
+        uc = enua.UserCache.getUserCache(user_id)
+        ma_entry_list = uc.getMessage(["background/motion_activity"], ma_time_query)
+    else: 
+        ma_entry_list = [] 
+    loc_time_query = estt.TimeQuery("data.ts", start_ts, end_ts)
+    loc_entry_list = list(estcs.find_entries(user_id, key_list=None, time_query=loc_time_query))
+    trip_time_query = estt.TimeQuery("data.start_ts", start_ts, end_ts)
+    trip_entry_list = list(ts.find_entries(key_list=None, time_query=trip_time_query))
+    place_time_query = estt.TimeQuery("data.enter_ts", start_ts, end_ts)
+    place_entry_list = list(ts.find_entries(key_list=None, time_query=place_time_query))
+    first_place_extra_query = {'$and': [{'data.enter_ts': {'$exists': False}},{'data.exit_ts': {'$exists': True}}]}
+    first_place_entry_list = list(ts.find_entries(key_list=None, time_query=None, extra_query_list=[first_place_extra_query]))
+    logging.info("First place entry list = %s" % first_place_entry_list)
+    combined_list = ma_entry_list + loc_entry_list + trip_entry_list + place_entry_list + first_place_entry_list
```

Comment:
with this change, the exported files are the same. This doesn't affect the functionality, but it does give us additional peace of mind.

```
$ diff /tmp/foo_576e37c7-ab7e-4c03-add7-02486bc3f42e /tmp/bar_576e37c7-ab7e-4c03-add7-02486bc3f42e
```

```suggestion
    combined_list = loc_entry_list + ma_entry_list + trip_entry_list + place_entry_list + first_place_entry_list
```